frontend/dynamic-demo-plugin/src/components/StatusComponents.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import {
+  Status,
+  GenericStatus,
+  ErrorStatus,
+  SuccessStatus,
+  PendingStatus,
+  ProgressStatus,
+  InfoIcon,
+  SuccessIcon,
+  ErrorIcon,
+  WarningIcon,
+  StatusIconAndText,
+} from '@console/dynamic-plugin-sdk/api';
+
+const StatusComponentsDemo: React.FC = () => {
+  return (
+    <div>
+      <h2>StatusComponents from Dynamic Plugin SDK</h2>
+      <div>
+        <Status title="status" status="status" />
+        <GenericStatus Icon={<div>icon</div>} title="status" />
```

Comment:
`<div>icon</div>` is not a valid `Icon`.

Fix the demo or update the `Icon` type.

Reply:
fixed